aries-site/src/pages/templates/forms.mdx

```diff
@@ -207,6 +208,27 @@ For standard forms, use an asterisk(\*) to indicate required field/controls.
   <CustomizeExample />
 </Example>
 
+### Character counter
+
+**When to use**  
+In cases where the user is given an opportunity to express themselves or would likely exceed a field's character limit. Examples include some description fields, feedback, feature requests, reporting issues, etc.  
+
+**Guidance**
+- Place the character counter at the bottom right of the field.  
+- [See guidance on character limits for fields](https://design-system.hpe.design/components/tag#anatomy). If you have doubts about how many characters to limit in the field, set a generous limit so as to not frustrate the user. When possible, check analytics to see how long it is taking your users to complete this task. Longer than usual time could indicate the character limits are too few. Increase accordingly.
+- Use a numerator/denominator format. Ex. 0/128.  
+- Count down the characters as the user is typing. Ex. 52/128 if the user has typed 52 characters. 53/128 when they type an extra character, etc.  
+- The character counter should be the same color as the field title.  
```

Comment:
We can just say "text" as the color

Reply:
Could you write the whole sentence? I didn't quite catch what you meant :)

Reply:
The character counter should be `text` color.

Reply:
- The character counter should use the `text` styling (same color, weight, font, etc.)

Reply:
`text` was referring solely to the color. If we are saying text styling in general, we can just say:

- The character counter should use default text styling.